Tasks/Drobyshev/Task3/task_lesson.py

```diff
@@ -0,0 +1,272 @@
+    print(a[12]*a[13], end="  ")
+    print(a[13]+a[14], end="  ")
+
+elif len(a) == 16:
+    print(a[0]*a[1], end="  ")
+    print(a[1]+a[2], end="  ")
+    print(a[2]*a[3], end="  ")
+    print(a[3]+a[4], end="  ")
+    print(a[4]*a[5], end="  ")
+    print(a[5]+a[6], end="  ")
+    print(a[6]*a[7], end="  ")
+    print(a[7]+a[8], end="  ")
+    print(a[8]*a[9], end="  ")
+    print(a[9]+a[10], end="  ")
+    print(a[10]*a[11], end="  ")
+    print(a[11]+a[12], end="  ")
+    print(a[12]*a[13], end="  ")
+    print(a[13]+a[14], end="  ")
+    print(a[14]*a[15], end="  ")
+
+elif len(a) == 17:
+    print(a[0]*a[1], end="  ")
+    print(a[1]+a[2], end="  ")
+    print(a[2]*a[3], end="  ")
+    print(a[3]+a[4], end="  ")
+    print(a[4]*a[5], end="  ")
+    print(a[5]+a[6], end="  ")
+    print(a[6]*a[7], end="  ")
+    print(a[7]+a[8], end="  ")
+    print(a[8]*a[9], end="  ")
+    print(a[9]+a[10], end="  ")
+    print(a[10]*a[11], end="  ")
+    print(a[11]+a[12], end="  ")
+    print(a[12]*a[13], end="  ")
+    print(a[13]+a[14], end="  ")
+    print(a[14]*a[15], end="  ")
+    print(a[15]+a[16], end="  ")
+
+elif len(a) == 18:
+    print(a[0]*a[1], end="  ")
+    print(a[1]+a[2], end="  ")
+    print(a[2]*a[3], end="  ")
+    print(a[3]+a[4], end="  ")
+    print(a[4]*a[5], end="  ")
+    print(a[5]+a[6], end="  ")
+    print(a[6]*a[7], end="  ")
+    print(a[7]+a[8], end="  ")
+    print(a[8]*a[9], end="  ")
+    print(a[9]+a[10], end="  ")
+    print(a[10]*a[11], end="  ")
+    print(a[11]+a[12], end="  ")
+    print(a[12]*a[13], end="  ")
+    print(a[13]+a[14], end="  ")
+    print(a[14]*a[15], end="  ")
+    print(a[15]+a[16], end="  ")
+    print(a[16]*a[17], end="  ")
+
+elif len(a) == 19:
+    print(a[0]*a[1], end="  ")
+    print(a[1]+a[2], end="  ")
+    print(a[2]*a[3], end="  ")
+    print(a[3]+a[4], end="  ")
+    print(a[4]*a[5], end="  ")
+    print(a[5]+a[6], end="  ")
+    print(a[6]*a[7], end="  ")
+    print(a[7]+a[8], end="  ")
+    print(a[8]*a[9], end="  ")
+    print(a[9]+a[10], end="  ")
+    print(a[10]*a[11], end="  ")
+    print(a[11]+a[12], end="  ")
+    print(a[12]*a[13], end="  ")
+    print(a[13]+a[14], end="  ")
+    print(a[14]*a[15], end="  ")
+    print(a[15]+a[16], end="  ")
+    print(a[16]*a[17], end="  ")
+    print(a[17]+a[18], end="  ")
+
+elif len(a) == 20:
+    print(a[0]*a[1], end="  ")
+    print(a[1]+a[2], end="  ")
+    print(a[2]*a[3], end="  ")
+    print(a[3]+a[4], end="  ")
+    print(a[4]*a[5], end="  ")
+    print(a[5]+a[6], end="  ")
+    print(a[6]*a[7], end="  ")
+    print(a[7]+a[8], end="  ")
+    print(a[8]*a[9], end="  ")
+    print(a[9]+a[10], end="  ")
+    print(a[10]*a[11], end="  ")
+    print(a[11]+a[12], end="  ")
+    print(a[12]*a[13], end="  ")
+    print(a[13]+a[14], end="  ")
+    print(a[14]*a[15], end="  ")
+    print(a[15]+a[16], end="  ")
+    print(a[16]*a[17], end="  ")
+    print(a[17]+a[18], end="  ")
+    print(a[18]*a[19], end="  ")
```

Comment:
we discussed it on the lesson, this is super not-optimal solution from the algorithmic point of view, such manual processing of all the variants is just ugly, but anyway, it is a solution, so I will grant you 30 additional points for it (and 100 for the task itself)